addons/links/src/manager.ts

```diff
@@ -4,7 +4,9 @@ import EVENTS, { ADDON_ID } from './constants';
 
 export const register = () => {
   addons.register(ADDON_ID, api => {
-    api.on(EVENTS.REQUEST, ({ kind, name }) => {
+    const channel = addons.getChannel();
+
+    channel.on(EVENTS.REQUEST, ({ kind, name }) => {
```

Comment:
channel.on & api.on should be the same fn in the manager. I'm wondering why api.on wasn't working.

Reply:
@ndelangen As am I. Not only wasn't it working, it was also undefined. Maybe it's because it's React Native? It's the only non-ondevice addon I have set up.